### DISCUSSION TITLE
licença

### DISCUSSION THREAD

**Discussion by fabiobom1** [aclap-dev/video-downloadhelper#1397] [License & Purchase requests]
### What's your issue?

já uso a muito tempo só que a agora não estou conseguindo ativá-lo . 
e não sei onde encontrar a minha licença

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You would be greatly helping yourself to click the `Copy information to clipboard` button in the VDH settings, then pasting that information in here.  Also, look in the Wiki.  There's a link for that at the top of every page of this forum.  There are chapters of the Wiki that deal with license issues.  You will find an E-mail address in that information.  You should probably send an E-mail to that address.  Include your clipboard information that I am telling you you should post here.

**Comment by jcfrog** [comment]
@fabiobom1 please send us an email at licenses@downloadhelper.net
